libpod/events/events.go

```diff
@@ -89,7 +89,11 @@ func (e *Event) ToHumanReadable(truncate bool) string {
 		}
 		humanFormat += ")"
 	case Network:
-		humanFormat = fmt.Sprintf("%s %s %s %s (container=%s, name=%s)", e.Time, e.Type, e.Status, id, id, e.Network)
+		if e.Status == "create" || e.Status == "remove" {
+			humanFormat = fmt.Sprintf("%s %s %s %s ", e.Time, e.Type, e.Status, e.Network)
```

Comment:
This does not match the format described in https://github.com/containers/podman/issues/24032, in particular it needs to log the id, name and driver